cityofaustin/atd-moped
# Re-implement geocoder and speed dial (#2)

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/10406
https://github.com/cityofaustin/atd-data-tech/issues/10404

Notable changes:
- Reworks the speed dial styles to be set with `makeStyles` and without the CSS file imported before
- Prunes some redundant and unused speed dial styles
- Adds some constants for Mapbox-style pixel widths/heights that we often apply to map controls
- Removes some geocoder style overrides that served to make it the same width as the "Add component" form that sits below it in the current Moped map (happy to revert some of this if it makes sense to make it wider)

## Testing
**URL to test:** 

https://md-10406-geocoder-search--atd-moped-main.netlify.app/

**Steps to test:**
1. Go to a new or existing project's **Map** tab
2. Use the geocoder search in the top left to search for "Austin City Hall" and select the result returned
3. In the bottom right corner, use the speed dial to switch from the **Streets** to the "Aerial" base map (Netlify deploy preview links are not on the approved NearMap API token referrer list so the tiles won't work, but you will see failed requests for them)
4. When **Streets** is selected, try different viewport width and see the height of the speed dial shift to keep the Mapbox copright text or mobile info icon in view. Super open to feedback on this since it maybe feels weird to have the controls shift, but this is what Mapbox does OOTB with their zoom controls so maybe it isn't that bad.

---
#### Ship list
